roles/satellite-clone/tasks/main.yml

```diff
@@ -60,7 +60,7 @@
   command: subscription-manager repos --enable rhel-7-server-satellite-{{ satellite_version }}-puppet{{ puppet_version }}-rpms
   when:
     - enable_repos
-    - puppet_version | int >= 4
+    - puppet_version|int >= 4 and puppet_version|int < 5
```

Comment:
Could you just say `puppet_version|int == 4`?

Reply:
Looking at how puppet_version is derived ... I think it will surely work!